docs/source/serving/load_balancing.rst

```diff
@@ -0,0 +1,78 @@
+.. _load_balancing:
+
+Load Balancing
+===============
+
+In scenarios where multiple GPUs or nodes are necessary to handle the model serving workload, it is advisable to initiate several vLLM server instances and distribute incoming requests amongst them using an HTTP load balancer.
+
+For instances where vLLM is deployed via Docker, the `Traefik HTTP load balancer <https://doc.traefik.io/traefik/>`_ can be employed to evenly distribute the load across multiple vLLM server instances.
+
+Below is a sample configuration:
+
+.. code-block:: yaml
+
+   services:
+     ### Traefik load balancer
+     http-load-balancer:
+       image: traefik:v2.10
+       command:
+         # static config
+         --providers.docker
+
+         --entrypoints.web.address=:80
+
+       volumes:
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+
+       restart: unless-stopped
+
+     ### vLLM servers
+     # 0
+     vllm-server-0:
+       # ... Configuration of vLLM server
+       image: vllm/vllm-openai:latest
+       command: --model mistralai/Mistral-7B-v0.1
+       volumes:
+         - ./huggingface_cache:/root/.cache/huggingface:rw
```

Comment:
Please comment this line. What is `./huggingface_cache`? It should be the user's model cache directory right?

Reply:
Yes. This line is copied from [here](https://docs.vllm.ai/en/latest/serving/deploying_with_docker.html)